build/stf-run-ci/tasks/setup_stf_from_bundles.yml

```diff
@@ -82,8 +82,8 @@
 
 - name: Deploy SGO via OLM bundle
   shell:
-    cmd: "{{ base_dir }}/working/operator-sdk run bundle {{__smart_gateway_bundle_image_path}} --pull-secret-name=pull-secret --ca-secret-name=registry-tls-ca --namespace={{ namespace }} --timeout 600s"
+    cmd: "{{ base_dir }}/working/operator-sdk run bundle {{__smart_gateway_bundle_image_path}} {% if pull_secret is defined %}--pull-secret-name=pull-secret --ca-secret-name=registry-tls-ca{% endif %} --namespace={{ namespace }} --timeout 600s"
```

Comment:
Not exactly happy with this. Considering putting contents above this into two `block` items with tags added to them. When run, have it `set_fact` that I can use to define `pull-secret-name` and `ca-secret-name` separately.